<launch>

  <node pkg="joy" type="joy_node" name="joystick"/>

  <node pkg="squirtle_teleop" type="squirtle_joystick" name="squirtle_joystick">
    <param name="scale_angular" value="1.5"/>
    <param name="scale_linear" value="0.5"/>
    <param name="axis_deadman" value="4"/>
    <param name="axis_linear" value="1"/>
    <param name="axis_angular" value="0"/>
    <remap from="teleop/cmd_vel" to="cmd_vel"/>
  </node>

</launch>
